templates/keyword.go

```diff
@@ -31,6 +31,11 @@ const Keyword = `<!DOCTYPE html>
 			</div>
 
 			<div class="home middle very relaxed page grid" id="main">
+				<div class="sixteenn wide center aligned centered column">
```

Comment:
Is the second "n" in `sixteenn` on purpose?

Reply:
It is not.  Will fix.